lib/intllib.lua

```diff
@@ -1,45 +1,80 @@
+-- intllib.lua
+-- should be copied from ../intllib/lib/intllib.lua   (2019-05-15)
+
+-- Since 5.0.0 the MT function `minetest.get_translator` is provided.
 
 -- Fallback functions for when `intllib` is not installed.
--- Code released under Unlicense <http://unlicense.org>.
-
--- Get the latest version of this file at:
---   https://raw.githubusercontent.com/minetest-mods/intllib/master/lib/intllib.lua
-
-local function format(str, ...)
-	local args = { ... }
-	local function repl(escape, open, num, close)
-		if escape == "" then
-			local replacement = tostring(args[tonumber(num)])
-			if open == "" then
-				replacement = replacement..close
-			end
-			return replacement
-		else
-			return "@"..open..num..close
-		end
-	end
-	return (str:gsub("(@?)@(%(?)(%d+)(%)?)", repl))
-end
+-- Code released under minetest-mods/intllib/LICENSE.md -- LGPLv2.1+
```

Comment:
Question: shouldn't the license here be `Unlicense`?
```suggestion
--[[ 
Code released under the GNU LGPLv2.1+ (https://github.com/minetest-mods/intllib/LICENSE.md)
```
(I'll keep it like this for now.)
Remove all `-` on the start of all the lines.